test/parallel/test-freeze-intrinsics.js

```diff
@@ -7,3 +7,28 @@ assert.throws(
   () => Object.defineProperty = 'asdf',
   TypeError
 );
+
+// Ensure we can extend Console
+{
+  const logs = [];
+  class ExtendedConsole extends console.constructor {
```

Comment:
What's the purpose of using`console.constructor` here? It is simply `Object`, since `console` is a namespace per the WHATWG spec.

Reply:
@joyeecheung `console.constructor === console.Console` does seem to hold actually... is that a bug? I've changed this to use `console.Console` rather.

Reply:
@guybedford 

> is that a bug?

I don't think so, you might be using an older version of node? This has been false since v12 at least.

Reply:
Ahh, yes I was testing it on an older version thanks.

Reply:
Thanks again @joyeecheung, there was actually an issue with the console test which I've fixed up now.